app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options depending on your situation.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-have-experience" class="category__nav-card">
+            <h2>If you have classroom experience</h2>
+
+            <p>
+                If you have already spent some time working in the classroom, you may be able to follow the assessment only route to show you meet QTS.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-want-to-change-career" class="category__nav-card">
+            <h2>If you want to change career</h2>
+
+            <p>
+            Use the experience you’ve gathered so far to inspire students and bring your skills and expertise to the classroom.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/other-training-you-can-get" class="category__nav-card">
+            <h2>Other training and experience you can get</h2>
```

Comment:
As before - I don't think we can class this as 'training'

It's about getting experience (and a taste of teaching)?

Reply:
I think the URL should be updated as well as the h2

Reply:
@valentine-carter - as previously, I don't think we should talk about these alternatives as 'training'

Reply:
If you are going with 'If you.....', maybe something like 

- 'If you want to get experience of teaching'
- 'If you want to get a taste of teaching'
etc

Reply:
I've flipped it round. Hopefully this is better now.